about-us.html

```diff
@@ -65,211 +69,198 @@ <h2 class="section-heading">THE STORY</h2>
             </dl>
             <ol class="about-us-list">
                 <li class="list-item">
-                    <h3 class="list-item-heading">HIGH QUALITY</h3>
+                    <h3 class="list-item sub-heading">HIGH QUALITY</h3>
                     <p class="list-item-number">01</p>
                     <p class="list-item-sub">Lorem Ipsum is simply dummy text of the printing and typesetting industry.
                     </p>
                 </li>
                 <li class="list-item">
-                    <h3 class="list-item-heading">BIG COLLECTION</h3>
+                    <h3 class="list-item sub-heading">BIG COLLECTION</h3>
                     <p class="list-item-number">02</p>
                     <p class="list-item-sub">Lorem Ipsum is simply dummy text of the printing and typesetting industry.
                     </p>
                 </li>
                 <li class="list-item">
-                    <h3 class="list-item-heading">TOP RESOURCE</h3>
+                    <h3 class="list-item sub-heading">TOP RESOURCE</h3>
                     <p class="list-item-number">03</p>
                     <p class="list-item-sub">Lorem Ipsum is simply dummy text of the printing and typesetting industry.
                     </p>
                 </li>
                 <li class="list-item">
-                    <h3 class="list-item-heading">BIG COMMUNITY</h3>
+                    <h3 class="list-item sub-heading">BIG COMMUNITY</h3>
                     <p class="list-item-number">04</p>
                     <p class="list-item-sub">Lorem Ipsum is simply dummy text of the printing and typesetting industry.
                     </p>
                 </li>
             </ol>
         </section>
         <section class="out-team">
-            <h2 class="section-heading">OUR TEAM</h2>
+            <h2 class="section heading">OUR TEAM</h2>
             <ol class="ot-members">
                 <li class="ot-member">
-                    <h3 class="tony-wings-heading">Jason Smith</h3>
-                    <img src="#" class="ot-portrait" alt="Jason Smith">
+                    <img src="images\portraits\jason-smith.webp" class="ot-portrait" alt="Jason Smith" width="120" height="120">
                     <div class="name-pos">
-                        <p class="ot-member-name">Jason Smith</p>
+                        <h3 class="member sub-heading">Jason Smith</h3>
                         <p class="ot-member-pos">Co - Founder</p>
                     </div>
                     <div class="ot-links">
-                        <a href="#" class="ot-linkedin">LinkedIn</a>
-                        <a href="#" class="ot-facebook">Facebook</a>
-                        <a href="#" class="ot-instagram">Instagram</a>
-                        <a href="#" class="ot-telegram">Telegram</a>
+                        <a href="#" target="_blank" class="ot-social"><img src="Images\icons\linkedin.svg" alt="linkedin icon" width="20" height="20"></a>

```

Comment:
When renaming a folder or directory, make sure to update all references to it throughout your project. This ensures that your files can be correctly located and linked. Missing one or two references can result in broken links or missing resources, leading to errors on your website.